types/react-tabs-redux/Tabs.d.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+export interface TabsProps {
+    name?: string;
+    onChange?: (selectedTab: string, name: string) => void;
+    handleSelect?: (tab: string, name: string) => void;
+    selectedTab?: string;
+    activeLinkStyle?: object;
+    visibleTabStyle?: object;
+    disableInlineStyles?: boolean;
+    renderActiveTabContentOnly?: boolean;
+}
+
+export class Tabs extends React.Component<TabsProps> {}
```

Comment:
I think you want to also add `ReactTabs` to global and add global test for same as documentation shows that it can be used as global https://github.com/patrik-piskay/react-tabs-redux#installation
Refer [this](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module) for more details.

Reply:
Hey @patrick-mackay, could you assist me with this? I tried to test UMD build https://codepen.io/anon/pen/qvwzve?editors=1111, but it says that noting is exported

Reply:
It doesn't work as expected, so I've just removed it from the readme. I don't think it'll be supported ever again.

Reply:
@sheetalkamat so, I think it's resoled for now